doc/classes/Object.xml

```diff
@@ -656,6 +656,7 @@
 			<description>
 				Returns [code]true[/code] if a metadata entry is found with the given [param name]. See also [method get_meta], [method set_meta] and [method remove_meta].
 				[b]Note:[/b] Metadata that has a [param name] starting with an underscore ([code]_[/code]) is considered editor-only. Editor-only metadata is not displayed in the Inspector and should not be edited, although it can still be found by this method.
+				[b]Note:[/b] A metadata’s [param name] must be a valid identifier. A valid identifier cannot start with a number.
```

Comment:
It could refer directly to the method used for the validation (its description tells what's valid). Also I'd replace the order of the notes as the valid identifier one is more general.
```suggestion
				[b]Note:[/b] A metadata’s [param name] must be a valid identifier as per [method StringName.is_valid_identifier] method.
				[b]Note:[/b] Metadata that has a [param name] starting with an underscore ([code]_[/code]) is considered editor-only. Editor-only metadata is not displayed in the Inspector and should not be edited, although it can still be found by this method.
```